webpack.config.js

```diff
@@ -0,0 +1,40 @@
+const path = require('path');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+
+module.exports = {
+  mode: 'development',
+  entry: './src/index.js',
+  devServer: {
+    static: './dist',
+  },
+  plugins: [
+    new HtmlWebpackPlugin({
+      template: './src/index.html',
+    }),
+  ],
+  output: {
+    filename: 'main.js',
+    path: path.resolve(__dirname, 'dist'),
+    clean: true,
+    publicPath: '/',
+  },
+  // optimization: {
+  //   runtimeChunk: 'single',
+  // },
```

Comment:
- [ ] [REQUIRED] Please, make sure that you don't use commented code in your code base because it makes the code base unclean and harder to maintain.